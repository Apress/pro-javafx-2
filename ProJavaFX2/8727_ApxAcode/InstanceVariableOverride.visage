/*
 * @test
 * @run
 */

class A {
  var i: Integer = 1024;
}

class B extends A {
  override var i = 2048;
}

var a = A {};
var b = B {};

println("a.i = {a.i}.");
println("b.i = {b.i}.");
