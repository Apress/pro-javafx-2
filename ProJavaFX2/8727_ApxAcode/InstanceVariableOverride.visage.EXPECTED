a.i = 1024.
b.i = 2048.
